### PR TITLE
Add workflow and process management commands

### PR DESCRIPTION
**Pull request by dfinteligenz**
## Description
Add workflow and process management commands

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [x] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/services/product_configuration/kai_product_configuration.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+)
+
+type ConfigProperty struct {
+	Key   string
+	Value string
+}
+
+type KaiProductConfiguration struct {
+	krt.Krt
+}
```
Comment (kafkaphoenix) [inline internal/services/product_configuration/kai_product_configuration.go:21]: this struct is missing some fields no?

Comment (dfinteligenz) [reply internal/services/product_configuration/kai_product_configuration.go:21]: Nop, when you add a struct inside another struct without a field name, the parent struct inherits all the fields from the `krt.KRT` struct.

Is the way inheritance works in Go.

**Review by kafkaphoenix — APPROVED** [review]
Amazing work!

**Review by kafkaphoenix — APPROVED** [review]

File: authserver/authserver.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+	as.logger.Info(fmt.Sprintf("Booting up the server at: %s", serverAddress))
+
+	as.closeApp.Add(1)
+
+	http.HandleFunc(fmt.Sprintf("/%s", as.config.EmbeddedServerConfig.CallbackPath),
+		func(w http.ResponseWriter, r *http.Request) {
+			as.logger.Info(fmt.Sprintf("Callback received: %v", r.URL))
+
+			code := r.URL.Query().Get("code")
+			if code != "" {
```
Comment (kafkaphoenix) [inline authserver/authserver.go:103]: unnest this logic please

**Review by kafkaphoenix — DISMISSED** [review]

**Review by kafkaphoenix — DISMISSED** [review]

**Review by kafkaphoenix — APPROVED** [review]

**Pull request merged**